Sources/Views/BatchUpdates.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 
 
 /// A set of updates to apply to a `ReloadableView`.
-public struct BatchUpdates {
+public class BatchUpdates: NSObject {
```

Comment:
Please add a comment explaining why NSObject base class is useful, maybe with an example usecase.